content/help/apiv2/smart-ids.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Smart IDs
+tags:
+- API
+- APIv2
+- API v2
+category: "apiv2"
+swagger: true
```

Comment:
I don't think this is supposed to be here. It adds the swagger js to the page.
```suggestion
```

Reply:
> It adds the swagger js to the page.

TIL